app/views/public_pages/template_index.html.erb

```diff
@@ -18,27 +18,48 @@
         <thead>
          <% if @templates.count > TABLE_FILTER_MIN_ROWS %>
             <tr>
-              <th colspan="3" class="sorter-false">
+              <th colspan="5" class="sorter-false">
                 <%= render(partial: "shared/table_filter",
                            locals: {path: public_templates_path, placeholder: _('Filter templates')}) %>
               </th>
             </tr>
           <% end %>
           <tr>
             <th><%= _('Template Name') %></th>
-            <th><%= _('Organisation Name') %></th>
             <th class="sorter-false text-center"><%= _('Download') %></th>
+            <th><%= _('Organisation Name') %></th>
+            <th class="sorter-false"><%= _('Funder Links') %></th>
+            <th class="sorter-false"><%= _('Sample Plans') %></th>
           </tr>
         </thead>
         <tbody>
           <% @templates.each do |template| %>
             <tr>
               <td><%= template.title %></td>
-              <td><%= template.org.name %></td>
               <td class="text-center">
                 <%= link_to _('DOCX'), template_export_path(template.dmptemplate_id, format: :docx), target: '_blank' %>
+                <br>
                 <%= link_to _('PDF'), template_export_path(template.dmptemplate_id, format: :pdf), target: '_blank' %>
               </td>
+              <td><%= template.org.name %></td>
```

Comment:
please, can you use the helper I wrote (https://github.com/DMPRoadmap/roadmap/blob/CDL-MVP/app/helpers/template_helper.rb) will reduce verbosity and it has test cases in place.